test/helper/TestFactory.js

```diff
@@ -333,6 +333,8 @@ export class TestFactory {
   async exposeCallingActors() {
     await this.exposeConversationActors();
     this.calling_repository = new CallingRepository(
+      // @ts-ignore
```

Comment:
why is this necessary?

Reply:
strict mode error, already was there because of this.message_repository, now that the order is changed betterer thinks there is a new error added but it is an already existing old one